.circleci/src/jobs/notify_release.yml

```diff
@@ -0,0 +1,25 @@
+# Create a release (with release notes) on github and send a mattermost notification.
+parameters:
+  production_delivery:
+    description: Whether builds are delivered to the production or beta lane of the play store.
+    type: boolean
+docker:
+  - image: cimg/node:19.1.0
+resource_class: small
+steps:
+  - prepare_workspace
+  - restore_environment_variables
+  - install_app_toolbelt
+  - run:
+      name: Create github release
+      command: echo "export RELEASE_ID='$(app-toolbelt v0 release create all ${NEW_VERSION_NAME} ${NEW_VERSION_CODE} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --production-release << parameters.production_delivery >> --should-use-predefined-release-notes false)'" >> ${BASH_ENV}
```

Comment:
as this is a boolean, the convention is:
if it is set, it is true
if it is not set, it is false

But you would have to also adjust the toolbelt to fix this.

Reply:
okay i probably didn't get the problem. If we don't pass anything or false its fine for the beta use case.
If we pass true what will happen very unlikely that the `pre` will not set?
Can you explain me the issue?

Reply:
The issue is to make a public interface more robust and following conventions.

Reply:
i switched to optional flags
https://github.com/digitalfabrik/app-toolbelt/pull/13
removed here the not needed flag `--should-use-predefined-release-notes false`